### DISCUSSION TITLE
Cannot remove dead node - tells me if it not on the failed list

### DISCUSSION THREAD

**Discussion by madasus** [zwave-js/zwave-js-ui#3943] [Q&A] [answered]
I've done some searches across other issues and i'm coming up without an answer.

I have a failed device which is no longer physically powered on (it died).  zwavejs won't let me remove the failed node as it says

INFO Z-WAVE: The node removal process could not be started due to the following reasons:
· Node 16 is not in the list of failed nodes (ZW0360) removeFailedNode undefined

The node has no deviceID or any data (0xXXXX 0xXXXX-0xXXXX) 

if I interview i see this

2024-10-11T17:38:48.324Z CNTRLR   [Node 016] Interview attempt 1/5 failed, retrying in 5000 ms...
2024-10-11 13:38:48.326 ERROR Z-WAVE: [Node 016] Interview FAILED: Attempt 1/5 failed
2024-10-11T17:38:57.533Z CNTRLR   [Node 016] Interview attempt 2/5 failed, retrying in 10000 ms...
2024-10-11 13:38:57.534 ERROR Z-WAVE: [Node 016] Interview FAILED: Attempt 2/5 failed
2024-10-11T17:39:11.782Z CNTRLR   [Node 016] Interview attempt 3/5 failed, retrying in 15000 ms...
2024-10-11 13:39:11.783 ERROR Z-WAVE: [Node 016] Interview FAILED: Attempt 3/5 failed
2024-10-11T17:39:30.991Z CNTRLR   [Node 016] Interview attempt 4/5 failed, retrying in 20000 ms...
2024-10-11 13:39:30.992 ERROR Z-WAVE: [Node 016] Interview FAILED: Attempt 4/5 failed
2024-10-11T17:39:55.219Z CNTRLR   [Node 016] Failed all interview attempts, giving up.
2024-10-11 13:39:55.220 ERROR Z-WAVE: [Node 016] Interview FAILED: Maximum interview attempts reached

but the node is still not marked as failed.  How can I mark it as failed?

running the latest versions of every component.

zwave-js-ui: 9.21.1.e8667de
zwave-js: 13.4.0

Debug Info

`{
  "id": 16,
  "name": "workshop old 1",
  "loc": "Workshop",
  "values": [],
  "groups": [],
  "neighbors": [],
  "ready": false,
  "available": true,
  "hassDevices": {},
  "failed": false,
  "inited": false,
  "eventsQueue": [
    {
      "time": "2024-10-11T17:33:56.933Z",
      "event": "interview failed",
      "args": [
        {
          "errorMessage": "Attempt 1/5 failed",
          "isFinal": false,
          "attempt": 1,
          "maxAttempts": 5
        }
      ]
    },
    {
      "time": "2024-10-11T17:34:07.357Z",
      "event": "interview failed",
      "args": [
        {
          "errorMessage": "Attempt 2/5 failed",
          "isFinal": false,
          "attempt": 2,
          "maxAttempts": 5
        }
      ]
    },
    {
      "time": "2024-10-11T17:34:21.869Z",
      "event": "interview failed",
      "args": [
        {
          "errorMessage": "Attempt 3/5 failed",
          "isFinal": false,
          "attempt": 3,
          "maxAttempts": 5
        }
      ]
    },
    {
      "time": "2024-10-11T17:34:45.603Z",
      "event": "interview failed",
      "args": [
        {
          "errorMessage": "Attempt 4/5 failed",
          "isFinal": false,
          "attempt": 4,
          "maxAttempts": 5
        }
      ]
    },
    {
      "time": "2024-10-11T17:35:09.820Z",
      "event": "interview failed",
      "args": [
        {
          "errorMessage": "Maximum interview attempts reached",
          "isFinal": true,
          "attempt": 5,
          "maxAttempts": 5
        }
      ]
    },
    {
      "time": "2024-10-11T17:38:44.079Z",
      "event": "value added",
      "args": [
        {
          "commandClassName": "Node Naming and Location",
          "commandClass": 119,
          "property": "name",
          "newValue": "workshop old 1",
          "propertyName": "name"
        }
      ]
    },
    {
      "time": "2024-10-11T17:38:44.080Z",
      "event": "value added",
      "args": [
        {
          "commandClassName": "Node Naming and Location",
          "commandClass": 119,
          "property": "location",
          "newValue": "Workshop",
          "propertyName": "location"
        }
      ]
    },
    {
      "time": "2024-10-11T17:38:44.081Z",
      "event": "interview started",
      "args": []
    },
    {
      "time": "2024-10-11T17:38:44.104Z",
      "event": "interview stage completed",
      "args": [
        "ProtocolInfo"
      ]
    },
    {
      "time": "2024-10-11T17:38:48.327Z",
      "event": "interview failed",
      "args": [
        {
          "errorMessage": "Attempt 1/5 failed",
          "isFinal": false,
          "attempt": 1,
          "maxAttempts": 5
        }
      ]
    },
    {
      "time": "2024-10-11T17:38:57.534Z",
      "event": "interview failed",
      "args": [
        {
          "errorMessage": "Attempt 2/5 failed",
          "isFinal": false,
          "attempt": 2,
          "maxAttempts": 5
        }
      ]
    },
    {
      "time": "2024-10-11T17:39:11.784Z",
      "event": "interview failed",
      "args": [
        {
          "errorMessage": "Attempt 3/5 failed",
          "isFinal": false,
          "attempt": 3,
          "maxAttempts": 5
        }
      ]
    },
    {
      "time": "2024-10-11T17:39:30.992Z",
      "event": "interview failed",
      "args": [
        {
          "errorMessage": "Attempt 4/5 failed",
          "isFinal": false,
          "attempt": 4,
          "maxAttempts": 5
        }
      ]
    },
    {
      "time": "2024-10-11T17:39:55.220Z",
      "event": "interview failed",
      "args": [
        {
          "errorMessage": "Maximum interview attempts reached",
          "isFinal": true,
          "attempt": 5,
          "maxAttempts": 5
        }
      ]
    }
  ],
  "status": "Unknown",
  "interviewStage": "ProtocolInfo",
  "priorityReturnRoute": {},
  "customReturnRoute": {},
  "customSUCReturnRoutes": [],
  "applicationRoute": false,
  "hexId": "0xXXXX 0xXXXX-0xXXXX",
  "dbLink": "https://devices.zwave-js.io/?jumpTo=0xXXXX:0xXXXX:0xXXXX:0.0",
  "productLabel": "Unknown product 0xXXXX",
  "productDescription": "0xXXXX",
  "manufacturer": "Unknown manufacturer 0xXXXX",
  "protocolVersion": 1,
  "nodeType": 1,
  "endpointsCount": 0,
  "endpoints": [
    {
      "index": 0,
      "label": "Root Endpoint",
      "deviceClass": {
        "basic": 3,
        "generic": 16,
        "specific": 0
      }
    }
  ],
  "supportsSecurity": false,
  "supportsBeaming": false,
  "isControllerNode": false,
  "isListening": true,
  "isFrequentListening": false,
  "isRouting": true,
  "keepAwake": false,
  "maxDataRate": 9600,
  "deviceClass": {
    "basic": 3,
    "generic": 16,
    "specific": 0
  },
  "lastActive": 1726406407441,
  "firmwareCapabilities": {
    "firmwareUpgradable": false
  },
  "protocol": 0,
  "deviceId": "undefined-undefined-undefined",
  "_name": "workshop old 1 (Workshop)",
  "statistics": {
    "commandsTX": 0,
    "commandsRX": 0,
    "commandsDroppedRX": 0,
    "commandsDroppedTX": 0,
    "timeoutResponse": 0
  },
  "prioritySUCReturnRoute": false,
  "errorReceive": false,
  "errorTransmit": false
}
`

**Comment by drjjr2** [comment]
Try pinging the node to see if it then marks it as failed. That usually works for me.

**Comment by madasus** [comment]
Unfortunately not - i've tried this a bunch of times.

**Reply by madasus** [reply]
It is always spinning and showing a red circle for PrototolInfo - maybe there is a bug where it won't mark an unidentified node as dead?

![image](https://github.com/user-attachments/assets/99664e3c-0058-4f94-8700-efd7bc4b47e6)

**Reply by robertsLando** [reply]
Please make a [driver log](https://zwave-js.github.io/zwave-js-ui/#/troubleshooting/generating-logs?id=driver-logs), loglevel `debug` and attach it here as a file (drag & drop into the text field).

**Comment by madasus** [comment]
Hi - here is the log.

It is Node 16

I did a ping - waited for that to come back as failed - then did a remove failed node

[zwavejs_2024-10-15.log](https://github.com/user-attachments/files/17378606/zwavejs_2024-10-15.log)

**Reply by robertsLando** [reply]
cc @AlCalzone

**Comment by AlCalzone** [comment, marked as answer]
Your controller is becoming unresponsive. Do an OTW update of the firmware to the latest version (1.50), which can be found on https://www.support.getzooz.com/kb/article/1158-zooz-ota-firmware-files/

You may want to wait until nothing is happening to avoid any interruptions. If the firmware update aborts, there is a setting called "bootloader only", which you can check to re-attempt the upgrade.

**Comment by madasus** [comment]
@AlCalzone That was it - after the FW update I was able to remove the node immediately.  I had only updated to 1.4 a few weeks ago so wasn't expecting another FW so soon.

Thank you!

**Reply by AlCalzone** [reply]
Yeah, some recent 800 series firmwares have been pretty bad.
